src/twisted/python/_url.py

```diff
@@ -528,9 +528,10 @@ def child(self, *segments):
 
         For example::
 
-            >>> (URL.fromText(u"http://localhost/a/b?x=y")
-                 .child(u"c", u"d").asText())
-            u'http://localhost/a/b/c?x=y'
+            >>> u = URL.fromText(u"http://localhost/a/b?x=y")
+            >>> v = u.child(u"c", u"d")
+            >>> print(v.asText())
+            u'http://localhost/a/b/c/d?x=y'
```

Comment:
> Is the policy that we use Py2 reprs then? I assume the answer must be yes I suppose.

No idea.  This docstring is a bit weird.  Not many Twisted docstrings contain this kind of example.  Reading this, I notice that it's still wrong - print wouldn't show the repr().